packages/web-app-external/tests/unit/__snapshots__/app.spec.ts.snap

```diff
@@ -41,6 +44,7 @@ exports[`The app provider extension should fail for unauthenticated users 1`] =
 exports[`The app provider extension should show a loading spinner while loading 1`] = `
 <main class="oc-height-1-1 oc-flex oc-flex-center oc-flex-middle">
   <h1 class="oc-invisible-sr">"exampleApp" app page</h1>
+  <app-top-bar-stub resource="undefined"></app-top-bar-stub>
```

Comment:
Kinda confused because I thought I'd be able to mock `data` within the wrapper, but couldn't get it to work properly

Reply:
maybe `wrapper.vm.resource = xyz ` before the snapshot works? But just a wild guess

Reply:
When `resource` is mocked via `mock` or `mockDeep` it will result in `undefined` in the snapshots unfortunately. Probably because those return proxy objects, but I can't give you the exact reasoning.

Reply:
Interesting, maybe we can fix that in the snapshot lib